Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -295,19 +295,23 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         float deltaCotTheta2 = deltaCotTheta * deltaCotTheta;
         float error;
         float dCotThetaMinusError2;
-        if (m_config.enableCutsForSortedSP) {
+        if (m_config.cotThetaSorting) {
           // if the error is larger than the difference in theta, no need to
           // compare with scattering
           if (deltaCotTheta2 - error2 > scatteringInRegion2) {
```

Comment:
i only noticed now that this check was behind the enableCutsForSortedSP/now is behind the cotThetaSorting. Isn't this changing the logic? or is that check performed elsewhere?